### PR TITLE
Draw Projects

### PR DESCRIPTION
**Pull request by ZasoGD**
##### Little script to create in IITC various separated projects using draw tools.

* Create a new _project_ and draw in it.
* Create another new _project_ and draw in it.
* You can switch between the projects and see only its drawn elements.

###### NB: It's not possible see all the '_layers projects_' simultaneously.

If you want you can rename the plugin or incorporate it to _draw tools_.

[screenshot](https://cloud.githubusercontent.com/assets/4642351/8190426/aebe0a9e-1462-11e5-87d4-cbb182921306.jpg)

### CODE REVIEW

**Comment by BondUniverse** [comment]
@jonatkins When will this be approved/rejected?  This is a functionality that I know a lot of OPs would absolutely LOVE!  I hope it gets approved soon!!!  So much potential!!!

**Comment by pleasantone** [comment]
+1 please

**Comment by VBelozyorov** [comment]
@jonatkins please consider to merge this ASAP. It's _musthave_ feature

File: plugins/draw-projects.user.js [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+window.plugin.drawProjects.projects = [];
+
+window.plugin.drawProjects.scanStorage = function() {
+  // Create drawtools default localstorage if it not exist
+  if(!window.localStorage[window.plugin.drawProjects.PREKEY]) {
+    window.localStorage[window.plugin.drawProjects.PREKEY] = '';
+  }
+
+  for(field in window.localStorage) {
+    if(field.includes(window.plugin.drawProjects.PREKEY)) {
```
Comment (McBen) [inline plugins/draw-projects.user.js]: "string.includes" requires chrome 41 or firefox 40

use >>    if(field.indexOf(window.plugin.drawProjects.PREKEY) !== -1) {

**Comment by Duwacell** [comment]
i was looking for this feature for a long time ! 100 Thanks

**Comment by ZasoGD** [comment]
The IITC project on github now is stopped. I close this pull. When IITC will start again I'll request for this plugin (which has since changed) and for others.

**Pull request closed without merging**